presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaTableLayoutHandle.java

```diff
@@ -36,9 +48,24 @@ public KafkaTableHandle getTable()
         return table;
     }
 
+    @JsonProperty
+    public long getOffsetStartTs()
+    {
+        return offsetStartTs;
+    }
+
+    @JsonProperty
+    public long getOffsetEndTs()
+    {
+        return offsetEndTs;
+    }
+
     @Override
     public String toString()
     {
-        return table.toString();
+        return toStringHelper(this)
+                .add("table", table.toString())
+                .add("offset_start_ts", offsetStartTs)
```

Comment:
s/offset_start_ts/startOffsetTimestamp/g
generally, we do not put underscore in toString

Reply:
Fixed.